otter/integration/config.sh-template

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+#
+# To run the integration tests, you'll need to have various environment
+# settings established.
+
+# This is the username used to authenticate against Identity.
+export AS_USERNAME=joe_user
+
+# This is the password used to authenticate against Identity.
+export AS_PASSWORD=joes_password
```

Comment:
Yes, he's expected to create his own config.sh file and run it prior to invoking trial tests.  How to prevent them from being committed?  I wish I had a good answer for that.  I don't know of a better way though.  I'd love feedback regarding best-practices on this.